contracts/StateRootInterface.sol

```diff
@@ -24,9 +24,9 @@ pragma solidity ^0.5.0;
 interface StateRootInterface {
 
     /**
-     * @notice Gets the block number of latest committed state root.
+     * @notice Gets the block number of latest anchored state root.
```

Comment:
`committed` is actually more appropriate here. The state root interface is *not* specific to the anchor.